latest.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ODS": "assessment_centre_locations_2020_09_22.csv"
```

Comment:
I think each directory should have its own separate "latest.json" file. If we have any metadata at this level (I'm not sure we need any at this stage), it should refer to full file paths from its own point of view e.g. "ODS/assessment_centre_locations_2020_09_22.csv" rather than expecting the reader to reconstruct the convention.

Reply:
@amb26, if each directory has its own "latest.json" file, is it still necessary to specify the file path such as "ODS/..." in it? Having the path in `latest.json` or a schema file would make sense when a file or files are not hardcopied to the community directory but being linked in from elsewhere. We haven't talked about this case.

If not considering to have the file path info, what do you think of making `latest.json` look like:
```
{
	"fileName": "assessment_centre_locations_2020_09_22.csv"
}
```

Reply:
Yes, that's correct, I think the filenames in latest.json should be relative to the directory holding it